src/config.rs

```diff
@@ -105,5 +105,30 @@ pub fn read_config() -> Result<Config, String> {
         }
     }
 
+    // ensure each method has only one param with inject=true
+    for method in &config.rpcs.methods {
```

Comment:
can you put this and the endpoint validation code into a `validate_config` function